.travis/before_install.sh

```diff
@@ -30,8 +30,8 @@
 [ -n "$BUILD_TARGET" ] || exit 0
 
 die() {
-	echo " *** ERROR: " $*
-	exit 1
+    echo " *** ERROR: " $*
```

Comment:
```suggestion
    echo " *** ERROR:  $*"
```

Reply:
I would prefer this PR to only fix indentation issue.